lib/Sema/TypeChecker.h

```diff
@@ -884,6 +884,16 @@ ProtocolConformanceRef conformsToProtocol(Type T, ProtocolDecl *Proto,
                                           DeclContext *DC,
                                           SourceLoc ComplainLoc = SourceLoc());
 
+/// Determine whether the given type could dynamically conform to the given protocol.
+/// Meaning it may have conditional requirements to the protocol.
```

Comment:
I'm not sure I understand the line `Meaning it may have conditional requirements to the protocol.` – perhaps it could be re-phrased as something like:

```
This is similar to \c conformsToProtocol, but returns \c true for cases where
the type could be dynamically cast to a conforming type, such as a non-final
class where a subclass conforms.
```

What do you think?

Reply:
Agh, sorry for that ... I'll adjust thank you :))